### ISSUE TITLE
Latest android update brought back drm protection

### ISSUE THREAD

**Issue by evillurker** [Genymobile/scrcpy#5393]
Not sure what happened but since I updated my android device, DRM is now active again on my streaming service, any chance to update what was broken to bring it back?

**Comment by rom1v** [comment]
Android >= 12? #2129 #36

**Comment by evillurker** [comment]
Yes. But it used to work now it doesnt, I tried using a phone with previous android version and still no go. Something changed.. it worked for weeks now..
